app/DoctrineMigrations/Version20170127093841.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+use Symfony\Component\DependencyInjection\ContainerAwareInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Added indexes on wallabag_entry.is_starred and wallabag_entry.is_archived
+ */
+class Version20170127093841 extends AbstractMigration implements ContainerAwareInterface
+{
+    /**
+     * @var ContainerInterface
+     */
+    private $container;
+
+    private $indexStarredName = 'IDX_entry_starred';
+    private $indexArchivedName = 'IDX_entry_archived';
+
+    public function setContainer(ContainerInterface $container = null)
+    {
+        $this->container = $container;
+    }
+
+    private function getTable($tableName)
+    {
+        return $this->container->getParameter('database_table_prefix').$tableName;
+    }
+
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        $entryTable = $schema->getTable($this->getTable('entry'));
+        $this->skipIf($entryTable->hasIndex($this->indexStarredName), 'It seems that you already played this migration.');
+        $this->skipIf($entryTable->hasIndex($this->indexStarredName), 'It seems that you already played this migration.');
```

Comment:
Apart the fact the line is duplicated, you should better check if both index are defined in just one line.
Same for the `down()` part